### PR TITLE
Post:Iteration Dev:Master

### PR DESCRIPTION
**Pull request by tony-gomes**
This is US16 - US25 post iteration merge from DEV to Master

84 passing tests

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -118,26 +118,42 @@ label + input {
   width: 200px;
 }
 
-.error_message {
+#error {
```
Comment (mikez321) [inline app/assets/stylesheets/application.css]: nice standard messages here

File: app/views/pet_show_applications/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h2>Applications for <%= @pet.name %></h2>
+
+<section>
+  <% if @pet.pet_applications.size > 0 %>
+    <% @pet.pet_applications.each do |application| %>
+      <%= link_to application.name, "/pet_applications/#{application.id}" %>
+    <% end %>
+  <% else %>
+    <h3>There are no applications for <%= @pet.name %> yet.</h3>
```
Comment (mikez321) [inline app/views/pet_show_applications/index.html.erb]: I like this interpolation here!

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -118,26 +118,42 @@ label + input {
...
   display: inline-block;
   float: left;
-  width: 50%;
+  width: 80%;
   margin-bottom: 20px;
   border-radius: 5px;
   text-align: left;
 }
 
-.well_done {
```
Comment (elguapogordo) [inline app/assets/stylesheets/application.css]: a regretful edit but the show must go on

File: app/views/pet_show_applications/index.html.erb (hunk 2 above)
Comment (elguapogordo) [reply app/views/pet_show_applications/index.html.erb]: thank you, sir!

**Comment by tony-gomes** [comment]
Great work gents!

**Pull request merged**